### DISCUSSION TITLE
(1) Thank you (2) Have a listen to what GrandOrgue is doing out in the wild!

### DISCUSSION THREAD

**Discussion by JSB2500** [GrandOrgue/grandorgue#1520] [Show and play organ]
Hello Everyone!

Firstly, a huge thank you to everyone involved in creating GrandOrgue. I had it more or less forced on me by a friend. Initially I wasn't much keen on mixing techonology with my sacred world of music, but I managed to download and install it, find a free-of-charge organ I liked (Friesach), and I even managed to build the software (in Ubuntu IIRC).

The same friend gave me a pedalboard (in 2020), and I've been using that along with an old Yamaha keyboard to create some great (IMHO) YouTube videos!

Here are some of them in case you are interested in what this great software has been used for:

• Bach BWV 542: https://youtu.be/ZS-xY8NGo2o
• Bach BWV 565: https://youtu.be/M18kEXdfzjo
• Bach BWV 592: https://youtube.com/shorts/s9NQGbjSjS0
• Bach BWV 625: https://youtu.be/nF-rJSs99vQ
• Bach BWV 785: https://youtu.be/abSEA5F2Oko
• Gigout - Scherzo: https://youtu.be/wvdjNFg_RTg
• Parry - Jerusalem: https://youtu.be/3x2w_QsVDBM
• Handel - Zadok the Priest: https://youtu.be/KGWFwOVZee0

I mostly do registration changes using the bottom manual octave to control generals (white notes) and couplers (black notes).

I also use two of the Yamaha pedals to increment and decrement the registration incrementer (sequencer?), with the bottom manual note to reset the incrementer. The Gigout Scherzo is a **really** good example of this.

Finally, I do a lot of two manual playing by splitting the Yamaha keyboard into regions, as here:
• Bach BWV 785: https://youtu.be/1LJBnKzsu8Y
• Bach BWV 645: https://youtu.be/t3uJ65KuDRk

I guess I have the most Frankenstein-monster-esque setup in existence, but I think I put it to good use!

Thanks again.

**Comment by JCL324** [comment]
This is awesome JSB! I just got the new Yamaha PSR-SX-920 and been trying to get it work with my Mac (M1) and GO. In my searching, I found this page and wow what an inspiration! I've been wanting Hauptwerk and all the associated hardware necessary but couldn't figure out how to put it all together. Seeing you play on a single keyboard and pedal board is all I really need! 

Just wondering if you have a complete list of all your setup, especially the audio parts. The sound is amazing on Youtube.
